android/src/com/mapswithme/maps/settings/AboutFragment.java

```diff
@@ -55,6 +55,7 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     setupItem(R.id.telegram, false, root);
     setupItem(R.id.facebook, false, root);
     setupItem(R.id.twitter, false, root);
+    setupItem(R.id.instagram,false, root);
```

Comment:
Missing space before false :)

Reply:
Ok, sir. I'll add space and revert it back.